launchable/test_runners/pytest.py

```diff
@@ -137,14 +132,41 @@ def _pytest_formatter(test_path):
 def record_tests(client, json_report, source_roots):
 
     def data_builder(case: TestCase):
+        props = case.child(Properties)
+        result = {}
+        if props is not None:
+            markers = []
+            marker: Dict[str, str] = {}
+            for prop in props:
+                if prop.name in marker:
+                    """
+                        Here is an example of an XML file.
+                        As you can see, all property tags are included within a single properties tag,
+                        so we identify each marker object by checking if a name attribute with the same value exists.
+                        ```
+                            <properties>
+                                <property name="name" value="parametrize" />
+                                <property name="args" value="('y', [2, 3])" />
+                                <property name="kwargs" value="{}" />
+                                <property name="name" value="parametrize" />
+                                <property name="args" value="('x', [0, 1])" />
+                                <property name="kwargs" value="{}" />
+                            </properties>
+                        ```
+                    """
+                    markers.append(marker.copy())
+                    marker.clear()
+                marker[prop.name] = prop.value
+            if len(marker) > 0:
+                markers.append(marker)
+            result["markers"] = markers
```

Comment:
Q. Why did you use `markers` as the field name?
I thought ‘properties’ (or annotations) would match the field name better.

Reply:
Sorry, please forget this comment, I checked only XML outputs. 
The user needs to add `@pytest.mark` if they want to add. Also I found pytest provides `--markers`  option
The --markers option always gives you a list of available markers:

Thus, the `makrers` of field name isn't bad.